docker-compose/docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+version: '3'
+
+networks:
+  monitor-net:
+    driver: bridge
+
+services:
+
+  surveyor:
+    image: synadia/nats-surveyor
+    container_name:  nats-surveyor
+    volumes:
+      - $NATS_SURVEYOR_CREDS:/etc/surveyor/SYS.creds
+    entrypoint: /nats-surveyor
+    command: -c $NATS_SURVEYOR_SERVER_COUNT -creds /etc/surveyor/SYS.creds -s $NATS_SURVEYOR_SERVERS
```

Comment:
not sure if nats handles spaces in url lists? but might be worth quoting the server list in case someone put a space in there

Reply:
Yep, thanks!